ts/src/binance.ts

```diff
@@ -12745,18 +12704,29 @@ export default class binance extends Exchange {
         } else {
             throw new BadRequest (this.id + ' fetchMarginModes () supports linear and inverse subTypes only');
         }
-        const assets = this.safeList (response, 'positions', []);
+        let assets = this.safeList (response, 'positions', []);
```

Comment:
hum `crossed` is not unified 
![image](https://github.com/user-attachments/assets/6bed3f33-e5f6-4a37-8524-b9e40a8e3d06)